compiler/rustc_codegen_gcc/src/intrinsic/mod.rs

```diff
@@ -117,12 +117,6 @@ impl<'a, 'gcc, 'tcx> IntrinsicCallMethods<'tcx> for Builder<'a, 'gcc, 'tcx> {
                     let func = unsafe { std::mem::transmute(simple.expect("simple")) };
                     self.call(self.type_void(), None, None, func, &args.iter().map(|arg| arg.immediate()).collect::<Vec<_>>(), None)
                 },
-                sym::likely => {
-                    self.expect(args[0].immediate(), true)
-                }
-                sym::unlikely => {
-                    self.expect(args[0].immediate(), false)
-                }
```

Comment:
There are similar match arms [here](https://github.com/rust-lang/rust/blob/fcb42b42d6fc078b15a34ed27b969fe2ba290989/compiler/rustc_const_eval/src/interpret/intrinsics.rs#L409) that can be cleaned up now.